### PR TITLE
chore: remove unnecessary code, adding the IoT retirement in README

### PR DESCRIPTION
**Pull request by suztomo**
Part of https://github.com/googleapis/google-cloud-python/issues/11048.

- chore: remove unnecessary files
- docs: retire message in README

### CODE REVIEW

**Comment by suztomo** [comment]
The samples are still referenced by documents. Adding do-not-merge tag.

**Comment by suztomo** [comment]
cl/555166720

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,5 @@
+:**NOTE**: **This github repository is archived. Google Cloud IoT Core is being retired in August 2023. For details, see https://cloud.google.com/iot-core. **
```
Comment (parthea) [inline README.rst:1]:
```suggestion
:**NOTE**: **This github repository is archived. Google Cloud IoT Core is retired as of August 2023. For details, see https://cloud.google.com/iot-core. **
```

**Review by parthea — APPROVED** [review]
LGTM, suggested minor change to README

**Comment by parthea** [comment]
Ready to merge once cl/555166720 is submitted

**Comment by suztomo** [comment]
I just submitted cl/555317637.

**Review by parthea — APPROVED** [review]

**Pull request merged**